actions/workflows/ngi_uu_workflow.yaml

```diff
@@ -89,11 +89,42 @@ workflows:
                 publish:
                     flowcell_name: <% task(get_flowcell_name).result.result %>
                 on-success:
-                    - download_samplesheet_mount: <% $.skip_download_samplesheet_mount = false %>
-                    - download_samplesheet_clarity: <% $.skip_download_samplesheet_mount = true %>
+                    - get_year
+
+            get_year:
+              action: core.local
+              input:
+                cmd: echo 20$(echo <% $.runfolder_name %> | cut -c1-2)
+              publish:
+                current_year: <% task(get_year).result.stdout %>
+              on-success:
+                - rsync_to_summary_host_before_demultiplexing
```

Comment:
Question: do these tasks need to be done sequentially or can they be started independently? I.e. `get_flowcell_name` can be followed by `get_year` *and* the `download_samplesheet...` tasks. Of course, in that case, nothing should be done for `on-success` of `create_lims_integration_links`. On the other hand, could be that we will start syncing temporary files from the bcl2fastq process...

Reply:
I think they could be started independently, if it does not mean that the `current_year` variable is only published to that branch of the workflow, but I doubt that. And like you said, we might end up syncing temporary files, so I don't know if it's worth it, especially since the summary sync is quite fast. It would however be a good idea to review the whole workflow and make it more efficient.

Reply:
I think that if we want to start them independently, we should use a specific file list with rsync to avoid side effects. On the other hand, I'm ok with leaving it as it is. Mostly, I just want to raise the point that we sometimes can leave it for the workflow engine to order tasks.